Profile

mihaillatyshov opened the pull request.

mihaillatyshov committed:
Profile

Rioskill committed:
Update Wild_Spirits.md

Протестировал создание и редактирование мероприятия

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+    ![sub missing](./screenshots/LM/002_sub_missing.png)
+  - Кнопка "Подписки" 
+    - Подчеркивается при наведении
+    - При клиике открывается [модальное окно подписок](#модальное-окно-подписок)
```
Опечатка
```suggestion
    - При клике открывается [модальное окно подписок](#модальное-окно-подписок)
```

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+    
+    ![sub missing](./screenshots/LM/002_sub_missing.png)
+  - Кнопка "Подписки" 
+    - Подчеркивается при наведении
```
Мб стоит добавить, что курсор становится поинтером, потому что это не везде так)

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+      
+      ![sub extra](./screenshots/LM/005_sub_extra.png)
+    - Длинные имена обрезаются, что не ломает верстку
+    - При наведении на подписку подсвечивается текс и обводится картинка
```
Опечатка
```suggestion
    - При наведении на подписку подсвечивается текст и обводится картинка
```

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+      - Является ссылкой
+    - ${\color{red} Картинка \ не \ меняется }$
+    - ${\color{red} Карточка \ не \ меняется }$
+    - Текс подсвечивается 
+  - При клике на актвную зону карточки открывается страница нужного пользователя. [Страница другого пользователя](#страница-другого-профиля)
+
+- Кнопка "Редактровать профиль" переключает на режим [редактирование профиля](#редактрованте-профиля)    
+
+
+## Страница другого профиля
+- Если не подписаны на польтователя, отображается кнопка "Подписаться"
+- Если подписаны на пользователя, отображается кнопка "Отписаться"
+- При клике на кнопку "Подписаться"
+  - Подписывает на пользователя 
+  - Кнопка меняется на "Отписаться"
+- При клике на кнопк "Отписаться"
```
Опечатки)
```suggestion
    - Текст подсвечивается 
  - При клике на активную зону карточки открывается страница нужного пользователя. [Страница другого пользователя](#страница-другого-профиля)

- Кнопка "Редактировать профиль" переключает на режим [редактирование профиля](#редактирование-профиля)    


## Страница другого профиля
- Если не подписаны на пользователя, отображается кнопка "Подписаться"
- Если подписаны на пользователя, отображается кнопка "Отписаться"
- При клике на кнопку "Подписаться"
  - Подписывает на пользователя 
  - Кнопка меняется на "Отписаться"
- При клике на кнопку "Отписаться"
```

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+- Если не подписаны на польтователя, отображается кнопка "Подписаться"
+- Если подписаны на пользователя, отображается кнопка "Отписаться"
+- При клике на кнопку "Подписаться"
+  - Подписывает на пользователя 
```
Я бы подробнее расписал, появляется ли подписка в профиле

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+    - Пользователь появляется в подписках в модальном окне и на странице профиля
+
+- На странице другого профиля
+  - ${\color{red} Кнопки \ статуса \ подписки \ отображаютя \ не \ для \ личного \ профиля \ (не \ дает \ подписаться \ на \ подписки \ других \ пользователей и пользователи на которых подписаны могут кнопку подписаться) }$
```
Забыл пробелы в латексе + непонятная формулировка в скобках

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+    - Если не подписаны на него
+      - Подписывает на пользователя  
+      - Пользователь появляется в подписках в модальном окне
+    - ${\color{red} Если \ подписаны \ на \ него, \ то \ ничего \ не \ происходит }$ 
```
Наверное, надо сказать, что для таких пользователей кнопка "подписаться" не должна быть активной.

CyTb-On-Fire committed:
report

Rioskill committed:
Update Wild_Spirits.md

Corrected misspells, added positive cases for event creation and place creation

Rioskill commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+    - Открывается диалоговое окно системы для выбора изображения
+    - Картинка меняется на новую, если выбрали картинку 
+    - Картинка не меняется, если не выбрана картинка
+    - ${\color{red} При \ выборе \ формата \ svg \ картика \ отображается \ до \ сохранения, \ после \ сохранения \ она \ исчезвет \ и \ показывается \ alt }$
```
```suggestion
    - ${\color{red} При \ выборе \ формата \ svg \ картика \ отображается \ до \ сохранения, \ после \ сохранения \ она \ исчезает \ и \ показывается \ alt }$
```

Rioskill replied:
Опечатка

Rioskill committed:
Update Wild_Spirits.md

Поправил вёрстку

CyTb-On-Fire commented on Wild_Spirits.md:
```diff
@@ -0,0 +1,126 @@
+    - Длинные имена обрезаются, что не ломает верстку
+    - При наведении на подписку подсвечивается текс и обводится картинка
+    - При клике на подписку открывается страница нужного пользователя. [Страница другого пользователя](#страница-другого-профиля)
+    - При количестве подписок больше 4 выводятся только 4 подписки 
```
Не расписано поведение при отсутствии подписок

mihaillatyshov committed:
fixes

Rioskill committed:
Добавил проверки на длины полей

Rioskill committed:
Добавил кейс с пустым полем тегов

Rioskill committed:
Устранил ошибки в латексе

reddiridabl666 reviewed (approved):
Апрув

CyTb-On-Fire committed:
latex fix

CyTb-On-Fire committed:
latex fix

CyTb-On-Fire committed:
latex fix

CyTb-On-Fire committed:
latex fix

reddiridabl666 committed:
Update Wild_Spirits.md

mihaillatyshov committed:
Merge branch 'Wild_Spirits' into Wild_Spirits

mihaillatyshov committed:
Merge pull request #4 from CyTb-On-Fire/Wild_Spirits

Event Filtering report

mihaillatyshov committed:
Merge branch 'Wild_Spirits' into Wild_Spirits

mihaillatyshov committed:
Merge pull request #3 from Rioskill/Wild_Spirits

Update Wild_Spirits.md
